android/src/main/java/io/github/droidkaigi/feeder/Initializers.kt

```diff
@@ -15,3 +17,13 @@ class FirebaseInitializer : Initializer<Firebase> {
         return emptyList()
     }
 }
+
+class ArborInitializer : Initializer<Unit> {
+    override fun create(context: Context) {
+        Arbor.sow(Seedling())
```

Comment:
It looks work fine 👍 
But, How do you think about initialize with `LogCatSeedling` instead of `Seedling` ? 👀 

Please refer below:
https://github.com/ToxicBakery/Arbor#api

![image](https://user-images.githubusercontent.com/5106629/109528167-c07f2180-7af7-11eb-8902-aebdfe41a3d0.png)